src/Domains/Connectors/PasoRapido/DataTransferObject/BillingDetail.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kanvas\Connectors\PasoRapido\DataTransferObject;
+
+use Spatie\LaravelData\Data;
+
+class BillingDetail extends Data
+{
+    public function __construct(
+        public readonly string $document,
+        public readonly bool $fiscalCredit,
+        public readonly string $invoice,
+        public readonly string $pdf,
+    ) {
+    }
+
+    public static function fromArray(array $data): self
```

Comment:
@jesusantguerrero in theory this is not needed

Reply:
It is because their api is a trash, combining english and spanish so, im parsing their spanish field to match mine